src/interactive-auth.js

```diff
@@ -318,6 +320,8 @@ InteractiveAuth.prototype = {
         try {
             const result = await this._requestCallback(auth, background);
             this._resolveFunc(result);
+            this._resolveFunc = null;
```

Comment:
Should we do the same when calling `_rejectFunc` below on line 387?

Reply:
oh yes, probably